common/shared.c

```diff
@@ -1021,6 +1021,9 @@ ssize_t ft_rx(struct fid_ep *ep, size_t size)
 	}
 	/* TODO: verify CQ data, if available */
 
+	/* Ignore the size arg. Otherwise posted receive wouldn't have enough space
+	 * when the benchmark tests increments the send data size. Instead we post
+	 * a size of the entire receive buffer. */
```

Comment:
Hmm. I'm not sure if I understand your question correctly. The size needs to be ignored here so that the tests don't have any issues.